### PR TITLE
KFSPTS-5516: review and update overlay files

### PR DESCRIPTION
**Pull request by bhutchinson**

### CODE REVIEW

**Comment by bhutchinson** [comment]
@cah292 I've been assigning my upgrade PRs to @jhulslander, but he's out today and it's a good idea to start involving more folks anyway. I apologize for the size of this commit. Some of the initial upgrade work has been done in bulk. There are spreadsheets attached to the sub-tasks of KFSPTS-5516 listing the files I reviewed and my approach for each. Please let me know if you have any questions. Thanks.

**Comment by kyleoliveira** [comment]
retest this please

File: src/main/resources/edu/cornell/kfs/vnd/cu-ojb-vnd.xml [hunk 1]
```diff
@@ -253,7 +253,99 @@
     </collection-descriptor>  -->
 </class-descriptor>
 
-<class-descriptor class="org.kuali.kfs.vnd.businessobject.VendorAddress" table="PUR_VNDR_ADDR_T">
+    <class-descriptor class="org.kuali.kfs.vnd.businessobject.VendorTaxDetail" table="PUR_VNDR_DTL_T">
```
Comment (cah292) [inline src/main/resources/edu/cornell/kfs/vnd/cu-ojb-vnd.xml]: Is VendorTaxDetail meant to share the same table as VendorDetail? Is that how it's currently set up in KFS 6?

Comment (bhutchinson) [inline src/main/resources/edu/cornell/kfs/vnd/cu-ojb-vnd.xml]: Yes, that's correct, per https://kualico.atlassian.net/browse/FINP-275.

**Comment by kyleoliveira** [comment]
retest this please

**Comment by kyleoliveira** [comment]
retest this please

**Comment by kyleoliveira** [comment]
retest this please

**Comment by cah292** [comment]
Changes appear to be good in general. However, the KFSPTS-5516 spreadsheets list 3 updated files that I do not recall seeing in this PR: ElectronicInvoiceReject.jsp, PurchaseOrder.jsp, and Requisition.jsp. Were there indeed changes to them that should be reflected in the PR?

Also, I see that the unit tests failed due to a missing table. Are you expecting the unit tests to pass for this PR?

Will wait for feedback on the above before merging.

**Comment by bhutchinson** [comment]
I actually updated those three JSPs as part of a prior commit against KFSPTS-5520, sorry that wasn't clear. 

Yes, at this point, I don't expect the unit tests to pass quite yet. Kyle is still working on getting that to work.

Thanks.

**Comment by cah292** [comment]
Ok, sounds good. Merging.

**Pull request merged**